src/utils/inAppBrowser.js

```diff
@@ -46,3 +50,19 @@ export const openInAppBrowser = async (url: string, options?: InAppBrowserOption
   reportOrWarn('InAppBrowser.isAvailable() returned false', null, 'warning');
   throw new Error('InAppBrowser.isAvailable() returned false');
 };
+
+export const openUrl = async (url: string | null) => {
+  if (url) {
+    await openInAppBrowser(url).catch(showServiceLaunchError);
+  } else {
+    showServiceLaunchError();
+  }
+};
+
+export const showServiceLaunchError = () => {
+  Toast.show({
+    message: t('toast.cryptoPurchaseLaunchFailed'),
```

Comment:
Method name and message does not relate, I would suggest either rename `showServiceLaunchError` to `showCryptoPurchaseLaunchError` or change the message

Reply:
Done.